src/mongo_client.ts

```diff
@@ -628,7 +628,7 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> implements
             this,
             new RunAdminCommandOperation(
               { endSessions },
-              { readPreference: ReadPreference.primaryPreferred, noResponse: true }
+              { readPreference: ReadPreference.primaryPreferred, moreToCome: true }
```

Comment:
Note that the description on the PR says we're using `writeConcern` instead of `noResponse`, so I had the expectation that we'd be setting `writeConcern` to `0` here and not setting `moreToCome`. If this is the true approach can we either update the PR description or change this to use `writeConcern`?

Reply:
`RunAdminCommandOperation` is an internal command, and it uses the `moreToCome` flag to run on commands which don't support `writeConcern`. 

The only user-facing API change is to `CommandOperationOptions`  which does already support `writeConcern` as an option, so users should be able to make the switch from using `noResponse` to `writeConcern` pretty easily.